docs/remote-k8s-access/README.md

```diff
@@ -0,0 +1,178 @@
+# Background
+K8ssandra Operator is capable of creating and managing a multi-datacenter Cassandra cluster that spans multiple Kubernetes clusters. The operator uses Kubernetes APIs to manage objects in remote clusters; therefore, it needs to create client connections with appropriate permissions to remote clusters.
+
+This document describes what is involved with creating and managing connections to remote clusters.
+
+When the operator starts up it queries for ClientConfig objects. A ClientConfig provides the necessary information for the operator to create client connections to remote clusters. 
+
+Before getting into details of using ClientConfigs we need to provide some background on service accounts, kubeconfig files, and secrets.
+
+## Service Account
+All pods are associated with a service account. If no service account is specified for a pod, then the default one will be used.
+
+**Note:** Kubernetes creates a default service account in each namespace.
+
+The api server authenticates a service account with a token that is stored in a secret. After authentication is done Kubernetes RBAC determine what operations the service account is authorized to perform.
+
+To learn more about this, see the official Kubernetes [docs](https://kubernetes.io/docs/reference/access-authn-authz/authentication/).
+
+A service account for the operator should be created in each cluster that the operator needs to access. This is done automatically as part of the operator installation.
+
+**Note:** The operator needs to be installed in each cluster to which it needs access.
+
+## Kubeconfig
+A kubeconfig file configures access to one or more Kubernetes clusters. Clients such as kubectl or the [client-go](https://github.com/kubernetes/client-go) library use kubeconfig files to access the api server.
+
+K8ssandra Operator uses kubeconfig files to create remote client connections.
+
+A `context` is a container object in a kubeconfig file that has three properties:
+
+* `cluster`
+* `user`
+* `namespace`
+
+`cluster` is a reference to a `cluster` object which declares the URL and the CA cert of the api server.
+
+`user` is a reference to a `user` object which specifies details to authenticate against the api server. This could for example include a client certificate and key or a bearer token.
```

Comment:
```suggestion
`user` is a reference to a `user` object which specifies details to authenticate against the API server. This could for example include a client certificate and key or a bearer token.
```